File: sprint5/By lessons/lesson8code4/lesson8code4.java
public Node bigLeftRotation(Node v) {
    // Правым ребёнком становится новый корень правого поддерева.
    v.right = smallRightRotation(v.right);
    // Возвращаем новый корень поддерева.
    return smallLeftRotation(v);
}

File: sprint5/By lessons/lesson8code4/lesson8code4.cs
Node BigLeftRotation(Node v) {
    // Правым ребёнком становится новый корень правого поддерева.
    v.right = SmallRightRotation(v.right);
    // Возвращаем новый корень поддерева.
    return SmallLeftRotation(v);
}
